web/vibe/web/web.d

```diff
@@ -80,11 +80,19 @@ import std.encoding : sanitize;
 		query/form fields, the following rules are applied:
 
 		$(UL
-			$(LI An array of values is mapped to
+			$(LI A dynamic array of values is mapped to
 				`<parameter_name>_<index>`, where `index`
-				denotes the zero based index of the array entry. The length
-				of the array is determined by searching for the first
-				non-existent index in the set of form fields.)
+				denotes the zero based index of the array entry. Any missing
+				indexes will be left as uninitialized. Arrays can also be
```

Comment:
Shouldn't this be "left to their `init` value" instead of "uninitialized"?

Reply:
Well, I consider that uninitialized :) But yes, I can change.

Reply:
fixed.